cmd/crio-status/main.go

```diff
@@ -59,7 +73,7 @@ func main() {
 			Usage: "the container ID",
 		}),
 		Name:  "containers",
-		Usage: "retrieve information about containers",
+		Usage: "Display detailed information about the provided container ID.",
```

Comment:
Does CRI-O generally uppercase the start of Usage statements?  I know the other project do not.

Reply:
It depends on what we want to have: We can also stick to lower-case beginning but I thought since we write sentences it should be fine.

Reply:
FWIW, I much prefer starting with uppercase, but wanted to ask the question in case @mrunalp or @rhatdan had a differing opinion.